src/wms/src/test/java/org/geoserver/wms/wms_1_3/CapabilitiesAllowedRequestTypesTest.java

```diff
@@ -0,0 +1,42 @@
+package org.geoserver.wms.wms_1_3;
+
+import java.util.Map;
+
+import org.custommonkey.xmlunit.XMLUnit;
+import org.custommonkey.xmlunit.XpathEngine;
+import org.geoserver.wms.WMSTestSupport;
+import org.junit.Assert;
+import org.junit.Test;
+import org.w3c.dom.Document;
+import org.w3c.dom.NodeList;
+
+public class CapabilitiesAllowedRequestTypesTest extends WMSTestSupport {
```

Comment:
Running tests in GeoServer is slow due to the expensive setup, please do not create a new test class for a single test, merge in an existing test with a similar topic instead. This seems like a good candidate:
https://github.com/geoserver/geoserver/blob/772e58a2d3bbba009f3ab980c2171c31f6bf3f9f/src/wms/src/test/java/org/geoserver/wms/wms_1_3/CapabilitiesIntegrationTest.java